Formula/gtk-vnc.rb

```diff
@@ -22,12 +23,20 @@ class GtkVnc < Formula
   depends_on "gtk+3"
   depends_on "libgcrypt"
 
-  # submitted upstream at https://gitlab.gnome.org/GNOME/gtk-vnc/merge_requests/4
+  # Fix configuration failure with -Dwith-vala=disabled
+  # Remove in the next release.
+  patch do
+    url "https://gitlab.gnome.org/GNOME/gtk-vnc/-/commit/bdab05584bab5c2ecdd508df49b03e80aedd19fc.diff"
+    sha256 "1b260157be888d9d8e6053e6cfd7ae92a666c306f04f4f23a0a1ed68a06c777d"
+  end
+
+  # Fix compile failure in src/vncdisplaykeymap.c
+  # error: implicit declaration of function 'GDK_IS_QUARTZ_DISPLAY' is invalid in C99
   patch :DATA
```

Comment:
Has this been reported?

Reply:
Not yet. I'll see if I have time today to create GNOME account and open an issue.

Reply:
You can sign in with your GitHub account.

Reply:
Opened https://gitlab.gnome.org/GNOME/gtk-vnc/-/issues/16

Reply:
Let's add that to the comment in a syntax-only PR when this PR is merged.

Reply:
#78019